common/smoke-test/smoke-tests.yml

```diff
@@ -49,9 +62,9 @@ jobs:
       - template: ../../eng/pipelines/templates/steps/common.yml
 
       - task: NodeTool@0
-        displayName: Use Node $(NodeVersion)
+        displayName: Use Node $(SmokeTestNodeVersion)
```

Comment:
I wanted to avoid naming overlaps with NodeVersion from common.yml imported above. If someone understands devops variable precedence better and thinks I can override it as NodeVersion, let me know.

Reply:
I think we can reuse the approach currently done in archtype-sdk-integration.yml to take advantage of common place to handle native dependency. My understanding is that we need this native dependency to run on node 8.

So you can set matrix variable name to NodeTestVersion and replace the task to set node version with below template.
- template: ../steps/use-node-test-version.yml

Reply:
Ah awesome, I didn't know about that template. Thanks for the pointer.

Reply:
I agree with @praveenkuttappan, I would use `NodeTestVersion` here.

Reply:
Updated.